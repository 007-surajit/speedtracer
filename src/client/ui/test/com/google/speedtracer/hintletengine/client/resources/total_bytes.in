# Input Record 1
# Expected: none
# This will load a single network event with 200000 bytes of data.  The minor
# alarm triggers at 500,000 bytes
{"type":2147483640,"time":1,"data":{"identifier":0,"timestamp":100,"dataLength":200000,"encodedDataLength":0},"sequence":1}

# Input Record 2
# Expected: HINTLET RECORD
# This record should put us past the 500,000 byte threshold
{"type":2147483640,"time":2,"data":{"identifier":0,"timestamp":100,"dataLength":600000,"encodedDataLength":0},"sequence":1}

# Input Record 3
# Expected: none
# This record should put us past the 1,000,000 byte threshold
{"type":2147483640,"time":3,"data":{"identifier":0,"timestamp":100,"dataLength":1000001,"encodedDataLength":0},"sequence":1}
