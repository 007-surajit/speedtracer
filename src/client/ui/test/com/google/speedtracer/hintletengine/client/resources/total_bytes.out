{"description":"600000 bytes downloaded, exceeds threshold of 500000 bytes.","hintletRule":"Total Bytes Downloaded","refRecord":2,"severity":3,"timestamp":2}
{"description":"1000001 bytes downloaded, exceeds threshold of 1000000 bytes.","hintletRule":"Total Bytes Downloaded","refRecord":3,"severity":2,"timestamp":3}
